emukit/core/acquisition/integrated_acquisition.py

```diff
@@ -0,0 +1,64 @@
+from typing import Union, Callable, Tuple
+
+import numpy as np
+
+from emukit.core.acquisition import Acquisition
+from emukit.core.interfaces import IModel, IPriorHyperparameters, IDifferentiable
+
+
+class IntegratedHyperParameterAcquisition(Acquisition):
```

Comment:
Sorry I am late to the party. The only minor question I have here is whether we should change the name of this file to conform to the class name

Reply:
We should indeed, I'll do that in a new PR